kotlin/apply-mosaic-rule-to-rasters/src/main/java/com/esri/arcgisruntime/sample/applymosaicruletorasters/MainActivity.kt

```diff
@@ -22,23 +22,37 @@ import android.util.Log
 import android.view.View
 import android.widget.AdapterView
 import android.widget.ArrayAdapter
+import android.widget.Spinner
 import androidx.appcompat.app.AppCompatActivity
 import com.esri.arcgisruntime.ArcGISRuntimeEnvironment
 import com.esri.arcgisruntime.layers.RasterLayer
 import com.esri.arcgisruntime.loadable.LoadStatus
 import com.esri.arcgisruntime.mapping.ArcGISMap
 import com.esri.arcgisruntime.mapping.BasemapStyle
 import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.MapView
 import com.esri.arcgisruntime.raster.ImageServiceRaster
 import com.esri.arcgisruntime.raster.MosaicMethod
 import com.esri.arcgisruntime.raster.MosaicOperation
 import com.esri.arcgisruntime.raster.MosaicRule
-import kotlinx.android.synthetic.main.activity_main.*
+import com.esri.arcgisruntime.sample.applymosaicruletorasters.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
   private val TAG = MainActivity::class.java.simpleName
 
+  private val activityMainBinding by lazy {
+    ActivityMainBinding.inflate(layoutInflater)
+  }
+
+  private val mapView: MapView by lazy {
+    activityMainBinding.mapView
+  }
+
+  private val spinner: Spinner by lazy {
+    activityMainBinding.spinner
+  }
+
   // create an image service raster from a url for an image service
   private val imageServiceRaster: ImageServiceRaster by lazy {
     ImageServiceRaster("https://sampleserver7.arcgisonline.com/arcgis/rest/services/amberg_germany/ImageServer").apply {
```

Comment:
This line returns a 404 error. Hence sample loads but fails to retrieve the `ImageServiceRaster`